buildspec-unit.yml

```diff
@@ -0,0 +1,8 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+    - pip install --upgrade pip --quiet
```

Comment:
quiet hides useful debug info

Reply:
I carefully try to keep only useful debug information. I did not consider upgrading pip useful. I am happy to remove if you have any specific considerations.